build.sbt

```diff
@@ -34,16 +22,13 @@ lazy val commonSettings = Seq(
   name := "firrtl",
   version := "1.4-SNAPSHOT",
   scalaVersion := "2.12.11",
-  crossScalaVersions := Seq("2.12.11", "2.11.12"),
-  addCompilerPlugin(scalafixSemanticdb),
-  scalacOptions := scalacOptionsVersion(scalaVersion.value) ++ Seq(
+  crossScalaVersions := Seq("2.13.2", "2.12.11", "2.11.12"),
+  scalacOptions := Seq(
     "-deprecation",
     "-unchecked",
     "-language:reflectiveCalls",
     "-language:existentials",
     "-language:implicitConversions",
-    "-Yrangepos",          // required by SemanticDB compiler plugin
-    "-Ywarn-unused-import" // required by `RemoveUnused` rule
```

Comment:
maybe unrelated too?

Reply:
Should be added back once we get `scalafix` working with `2.13`

Reply:
fixed in https://github.com/freechipsproject/firrtl/pull/1796/commits/9250cd688abb482a9c184f3cbf3bda172ac47050 and https://github.com/freechipsproject/firrtl/pull/1796/commits/005dd2519ade36a867d5b2dbaa199117860e14ca